lib/timeline/Core.js

```diff
@@ -670,7 +670,7 @@ class Core {
     const customTime = new CustomTime(this.body, util.extend({}, this.options, {
       time : timestamp,
       id,
-      snap: this.itemSet.options.snap
+      snap: this.options.snap
```

Comment:
Oops, this change makes the snapping functionality broken.

Reply:
fixed: a57d0a6

Reply:
I'm not sure this is still good.
The default behavior should use `this.itemSet.options.snap` so I think it is important to add that if `this.options.snap` doesn't exist, it should use `this.itemSet.options.snap`.
In the case of graph2D, a different solution should be given.

Reply:
Thank you for the review and sorry for the delay.

> In the case of graph2D, a different solution should be given.

I have two questions

* Could you share why you think a different solution should be given in the case of Graph2D with me?
* Do you have any good solutions?

Reply:
I would simply do something like:
``` javascript
snap: this.itemSet ? this.itemSet.options.snap : this.options.snap
```

Reply:
I see.
Applied your suggestion: 157d0a6